cxx-sensors/src/main/java/org/sonar/cxx/sensors/coverage/TestwellCtcTxtParser.java

```diff
@@ -79,113 +71,88 @@ public TestwellCtcTxtParser() {
   @Override
   public void processReport(final SensorContext context, File report, final Map<String, CoverageMeasures> coverageData) {
     LOG.debug("Parsing 'Testwell CTC++' textual format");
-    
+
     try {
       this.scanner = new Scanner(report).useDelimiter(SECTION_SEP);
```

Comment:
I missed this during the previous comment: in order to avoid this.scanner.close() calls (e. g. in case of exception), one could use so called "try with resources".
```JAVA
try ( this.scanner = new Scanner(report).useDelimiter(SECTION_SEP) )
{
    ...
}
catch ( ... )
{
}
```

Since Scanner is [AutoCloseable](https://docs.oracle.com/javase/8/docs/api/java/lang/AutoCloseable.html) it will be automatically closed when control flow lefts the try scope (see [documentation](https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html))